### PR TITLE
LKFT: KVM: update kvm-unit-tests repo link and sha

### PR DESCRIPTION
**Pull request by nareshkamboju**
LKFT: KVM: update kvm-unit-tests repo link and sha

Signed-off-by: Naresh Kamboju <naresh.kamboju@linaro.org>

### CODE REVIEW

File: testcases/master/template-kvm-unit-tests.yaml.jinja2 [hunk 1]
```diff
@@ -21,5 +21,5 @@
       parameters:
         SKIP_INSTALL: 'true'
         SMP: {% if DEVICE_TYPE == "juno-r2" %}'false'{% else %}'true'{% endif %}
-        GIT_REF: "{{KVM_UNIT_TESTS_REVISION | default('4b74c718c57d1697e3228e2c699b0fe9c1d24e97')}}"
+        GIT_REF: "{{KVM_UNIT_TESTS_REVISION | default('a3d5d6b9333e10898852b96311c3d3db0baf5a4e')}}"
```
Comment (roxell) [inline testcases/master/template-kvm-unit-tests.yaml.jinja2:24]: Its not required to change the sha in order to update the repository link.
Maybe this should be in its own directory.

**Comment by roxell** [comment]
The shortlog should say something like: "testcases: template-kvm-unit-test: change repository url"

File: testcases/master/template-kvm-unit-tests.yaml.jinja2 (hunk 1 above)
Comment (nareshkamboju) [reply testcases/master/template-kvm-unit-tests.yaml.jinja2:24]: I would like to fetch the latest test code.
The old git ref sha one is from June this new sha is from July 31.

**Comment by nareshkamboju** [comment]
@roxell Thanks for the review.
I have updated the commit short log.

**Comment by roxell** [comment]
LGTM, Thank you @nareshkamboju

**Pull request merged**